addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -6,7 +6,7 @@
                 <div class="d-flex align-items-start gap-2 bg-inherit">
                     <span class="o_avatar pt-1 position-relative o_card_avatar bg-inherit">
                         <img t-if="props.id"
-                            t-attf-src="/web/image/res.users/{{props.id}}/avatar_128"
+                            t-attf-src="/web/image/{{props.relation}}/{{props.id}}/avatar_128"
```

Comment:
This could use [imageUrl](https://github.com/odoo/odoo/blob/saas-18.3/addons/web/static/src/core/utils/urls.js#L71) helper, with a unique param if one is available.